### PR TITLE
feat: added login page

### PR DESCRIPTION
**Pull request by nakulsharma15**
Created Login page for e-commerce
index-1
index-2

### CODE REVIEW

**Review by sadathshariff — APPROVED** [review]
The Code looks Clean and Neat

File: pages/login.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="../style.css">
+
+    <link rel="stylesheet" href="../stylesheets/login.css">
+
```
Comment (shuntzhere) [inline pages/login.html]: Add this to the style.css file

**Review by shuntzhere — APPROVED** [review]
The code looks, just make sure you indent the code.

File: pages/login.html (hunk 1 above)
Comment (nakulsharma15) [reply pages/login.html]: Just did in the latest commit. Thanks for suggestion

**Pull request merged**